recipes/cfgfile/all/test_package/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ add_executable(${PROJECT_NAME}
     "${CMAKE_CURRENT_BINARY_DIR}/cfg.hpp"
 )
 
-target_link_libraries(${PROJECT_NAME} cfgfile)
+target_link_libraries(${PROJECT_NAME} cfgfile::cfgfile)
```

Comment:
The previous 0.2.9.0 recipe contained logic to recreate a `cfgfile` target (`_create_cmake_module_alias_targets`).
Is this removal intended?

Reply:
Yes. I don't see a need for it. But very old versions of `cfgfile` (even before Conan packaging) used its name without namespace. What do you think?

Reply:
You're the author of this library, so you're the authority.
It makes sense to remove it to get rid of old cruft.
Otoh, cmake has a property for that: [`DEPRECATION`](https://cmake.org/cmake/help/latest/prop_tgt/DEPRECATION.html).

Reply:
Let it be so, the user of this library should use `cfgfile::cfgfile` target. I do so in all my libraries. I believe that it's minor change for that users who already use `cfgfile` with target without namespace. `DEPRECATION` was introduced in CMake 3.17, but I prefer to support 3.16, as it's default CMake in Ubuntu 20.04.